backend/app/api/endpoints/work_queue.py

```
@@ -0,0 +1,299 @@
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=str(e))
+    except ConflictException as e:
+        raise HTTPException(status_code=status.HTTP_409_CONFLICT, detail=str(e))
+
+
+# ==================== Queue Messages ====================
+
+
+@router.get("/{queue_id}/messages", response_model=QueueMessageListResponse)
+async def list_queue_messages(
+    queue_id: int,
+    status: Optional[QueueMessageStatus] = Query(None),
```

Ruff (0.15.6)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)